Makefile

```diff
@@ -74,14 +87,14 @@ endif
 
 all: examples test build
 
-build: $(ARLIB)
+build: $(ARLIB) $(SOLIB)
```

Comment:
I'd like to avoid building the shared library by default, since its a special case - the default expectation is that pg_query is used as a static library, and examples/etc are written with that in mind.

I'd recommend we make a separate `build_shared` (or similar) target, and install depends on that as well as `build`.

Reply:
Resolved by 931a0b5038e2f9d869d7989c53b21f09509e6e3e.